jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/ConfigsTest.java

```diff
@@ -19,11 +19,13 @@
 import lombok.Getter;
 import org.apache.commons.lang3.RandomStringUtils;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
-
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
This is not needed unless you use Mockito specific annotations.

In general we prefer not to use specific runners (this would also ease the transition to JUnit5)